RELEASE_NOTES.md

```diff
@@ -12,6 +12,7 @@
 
 - A new module `frequenz.client.dispatch.test` has been added, providing a fake Service and Client as well as a `DispatchGenerator` to generate `Dispatch` instances filled with random data.
 - The `DispatchGenerator.generate_dispatch` method now accepts a `microgrid_id` parameter to generate `Dispatch` instances with a specific microgrid ID.
+- The `Client.create()` method now returns by default the newly created `Dispatch` object. This behavoir can be changed using the `fetch_resulting_dispatch` parameter.
```

Comment:
If the whole module was added in this release, there is no need to document changes to unreleased code, I think having only the "A new module..." entry is enough for this release.

Reply:
Hu, no, `Client.create()` is an existing core method

Reply:
Oh, OK. But still there is an entry above documenting a change to the test module right? Unrelated to this PR though.

Reply:
Ah yes. I see what you mean. I suppose we'll fix that with the release then.